src/main/java/com/example/Student/Controller/StudentController.java

```diff
@@ -155,6 +156,21 @@ public ResponseEntity<Object> addOneStudent(@PathVariable int id ,@RequestBody S
 		
 	}
 	
```

Comment:
Try to create a new column that doesnt exist in ur table. Also create separate rest control for that.It can be get or post anything.